constants/src/lib.rs

```diff
@@ -159,6 +159,11 @@ pub const CONNECTION_NOT_OPEN: OdbcState<'static> = OdbcState {
     odbc_3_state: "08003",
 };
 
+pub const INVALID_AUTHORIZATION_SPECIFICATION: OdbcState<'static> = OdbcState {
+    odbc_2_state: "28000",
+    odbc_3_state: "28000",
+};
+
```

Comment:
It looks like we missed this error https://learn.microsoft.com/en-us/sql/odbc/reference/appendixes/appendix-a-odbc-error-codes?view=sql-server-ver15